### PR TITLE
Gomodify

### PR DESCRIPTION
**Pull request by jefferai**
Each commit in this series is a single step starting with the current govendor, with the commit message explaining the specific change.

### CODE REVIEW

**Comment by i0rek** [comment]
Good point @rboyer! That would make it easier for ppl(like me) who are still running everything in their gopath.

**Comment by jefferai** [comment]
@rboyer @mkeeler @i0rek I've added three more commits with descriptions of each change based on feedback. PTAL!

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,18 @@
 language: go
 
 go:
-  - 1.10.x
-  - 1.11.x
+  - 1.11.4
```
Comment (rboyer) [inline .travis.yml]:
```suggestion
  - "1.11.4"
```
I think the quotes are necessary now to avoid yaml weirdness.

File: GNUmakefile [hunk 2]
```diff
@@ -46,12 +46,12 @@ tools::
 
 # updatedeps installs all the dependencies needed to test, build, and run
 updatedeps:: tools
-	govendor list -no-status +vendor | xargs -n1 go get -u
-	govendor update +vendor
+	go get -u
```
Comment (rboyer) [inline GNUmakefile]: Should this also have `./...`?

Comment (jefferai) [reply GNUmakefile]: Nope!

File: .travis.yml (hunk 1 above)
Comment (jefferai) [reply .travis.yml]: Ah really? Huh.

**Comment by rboyer** [comment]
I see you nuked `vendor`. I can understand that for libraries in a go-mod world. Was the overall plan to retain the `vendor` directory in deployables for the short term until the caching proxies are ready?

**Comment by jefferai** [comment]
@rboyer I'll put it back for now, tied to gomod in the `updatedeps` target.

**Pull request merged**